donut/modules/core/helpers.py

```diff
@@ -28,6 +28,9 @@ def get_member_data(user_id, fields=None):
     # Build the SELECT and FROM clauses
     s = sqlalchemy.sql.select(fields).select_from(sqlalchemy.text("members"))
 
+    # Build the WHERE clause
+    s = s.where(sqlalchemy.text("user_id = " + str(user_id)))
```

Comment:
+1, good catch. String concatenation like this is the easiest way you can introduce SQL injection vulnerabilities into your system.